### PR TITLE
(Publish on 10/7/2024 with attribute dictionary changes) Browser agent: add user action docs

### PR DESCRIPTION
**Pull request by metal-messiah**
This PR adds a general description of a new event type the Browser Agent is about to start capturing.

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "User Actions"
+metaDescription: "Understand where your users are clicking, scrolling, and interacting with your web app using User Actions in New Relic Browser."
+freshnessValidatedDate: 2024-04-23
+---
+
+User Actions in New Relic Browser provide insights into how users interact with your web app. You can see where users are clicking, scrolling, and interacting with your web app, and use this data to optimize your app's performance and user experience.
+
+* **Understand user behavior**: Identify the most used features and areas of your web app, and how users link actions together.
+* **Improving the user experience**: Discover areas of your web app that are causing users frustration. For example, you might see that users are getting are stuck on a particular page element or form field.
```
Comment (ptang-nr) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx:10]:
```suggestion
* **Improving the user experience**: Discover areas of your web app that are causing users frustration. For example, you might see that users are getting stuck on a particular page element or form field.
```

File: src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+User Actions currently encapsulate the following events:
+
+* auxclick
+* blur (window only)
+* click
+* copy
+* focus (window only)
+* keydown
+* paste
+* scrollend
```
Comment (ptang-nr) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx:27]: Not sure if we wanted to call out that scrollend doesn't consider the DOM target or it's too low-level for this doc.

Comment (metal-messiah) [reply src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx:26]: I think that may be too low-level here

**Review by rhetoric101 — DISMISSED** [review]
@metal-messiah thanks for adding this doc!

**Comment by akristen** [comment]
Hi @metal-messiah ! I shuffled the information to group together in a more docs style. I also converted your written intro to be more expository, rather than following our marketing what's new post pattern. I think it contains the info you want it to contain while maintaining a docs posture. :) I also removed the steps component and made the doc a bit more conversational, since the procedures are optional prereq checking and then using a simple NRQL query.

File: src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: "User actions"
+metaDescription: "Understand where your users are clicking, scrolling, and interacting with your web app using user actions in New Relic Browser."
+freshnessValidatedDate: 2024-04-23
+---
+
+User actions in browser monitoring helps you understand user behavior, like where users click, scroll, and interact with your web app. You can use user actions to link actions together, helping you gain insights into where your web app creates friction and increases user frustration. 
+
+User actions are tracked by default for Pro and Pro+SPA agents, but unavailable for the Lite browser agent. Confirm you're using the Pro or Pro+SPA agent by going to <DNT>**[one.newrelic.com > All Capabilities](https://one.newrelic.com/all-capabilities) > Browser**</DNT> (select your browser app) > <DNT>**Application settings**</DNT>. From this page, check that  <DNT>**Pro**</DNT> or <DNT>**Pro + SPA**</DNT> browser agent is selected.
```
Comment (akristen) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/user-actions.mdx:9]:
```suggestion
User actions are tracked by default for Pro and Pro+SPA agents, but unavailable for the Lite browser agent. We recommend that you first confirm you're using the Pro or Pro+SPA agent: 

Go to <DNT>**[one.newrelic.com > All Capabilities](https://one.newrelic.com/all-capabilities) > Browser**</DNT> (select your browser app) <DNT>**> Application settings**</DNT>. From this page, check that  <DNT>**Pro**</DNT> or <DNT>**Pro + SPA**</DNT> browser agent is selected. 
```

**Review by akristen — APPROVED** [review]
LGTM!

**Pull request merged**